src/helpers/validator.js

```diff
@@ -0,0 +1,56 @@
+import { check } from 'express-validator';
+
+/**
+ *  A class that provide static method to validate user input
+ */
+class Validator {
+  /**
+   *  @module isValidEmail
+   *  @function
+   *  @param {String} field
+   *  @returns {Object} - Express-validator
+   */
+  static isValidEmail(field = 'email') {
```

Comment:
Is there a reason why the `field` variable was initialized with the string `email`?

Reply:
Yes, normally the field should be an email so that you can just call Validator.isValidEmail() but in some cases, the field name may be different for example "secondary_email". To make the method reusable in such situation and to keep the advantage of not passing a field name when validating field name of email that's the reason for the initialization

Reply:
I'm not sure I understand the reason, but from what I can see, the `isValidEmail` method is reusable as-is without initializing the parameter, which for now is redundant.
For example, if I replace the string `email` with another string `xmailx`, the method still works fine.

Reply:
Yes! but in most case, the field is email and there is not memory or speed implication. It is just like a placeholder fully supported by javascript
It makes the code cleaner and gives the user more option. I see nothing wrong in it at all

Reply:
Maybe I'm making my points clear enough, but OK.